server/manager/src/main/java/org/apache/accumulo/manager/tableOps/compact/CleanUp.java

```diff
@@ -74,16 +75,21 @@ public long isReady(FateId fateId, Manager manager) throws Exception {
 
     try (
         var tablets = ample.readTablets().forTable(tableId).overlapping(startRow, endRow)
-            .fetch(PREV_ROW, COMPACTED).checkConsistency().build();
+            .fetch(PREV_ROW, COMPACTED, USER_COMPACTION_REQUESTED).checkConsistency().build();
         var tabletsMutator = ample.conditionallyMutateTablets(resultConsumer)) {
 
       t1 = System.nanoTime();
       for (TabletMetadata tablet : tablets) {
         total++;
         if (tablet.getCompacted().contains(fateId)) {
```

Comment:
This is completely unrelated to this PR, but another comment made me realize that certain compaction metadata is completely unexpected at this point.

```suggestion
        Preconditions.checkState(ensure selected column does not exists)
        if (tablet.getCompacted().contains(fateId)) {
```

Reply:
Should we put this into a follow on PR?